03-meca-notebooks.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Notebooks in MECA
+date: 2022/12/12
+---
+
+The goal of this article is to recommend community standards for including Notebooks in MECA files. This is intended to be complementary to our proposal regarding the [inclusion of computational notebooks within JATS XML files](https://github.com/curvenote/notebooks-now/blob/main/02-notebooks-as-jats.md).
+
+::: callout-note
```

Comment:
Is this valid quarto syntax?! Was expecting `{.callout-note}`?

Reply:
That is actually valid syntax as well (it is interpreted by Pandoc as a div with a class in this case), but I'll correct anyway as its probably confusing...

Reply:
Good to know. :)